_guides/defining_execution_frequency_for_flows.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Defining execution frequency for flows
+layout: article
+section: Building integration flows
+category: component
+order: 4
+since: 20181218
+---
+
+Scheduling tasks are part of the {{site.data.tenant.name}} functionality. Every flow is executed every 3 minutes which is Platform's default setting. Depending on the business needs, that default setting might not always be suitable. To proceed further, you may take advantage of some of the flow scheduling features that are going to be described in this article.
+
+## The Scheduling
+
+Please be aware, that changing the flow's schedule means modifying the flow. Here some of the requirements to follow: 
+
+*   For modifying a flow, you should have specific permissions.
+*   As soon as you modify the flow's schedule, you should restart it to let the System apply the changes.
+
+For more information regarding the CRON scheduling, please follow the steps below:
+
+### Navigation
```

Comment:
ATM moment using `###` does not work as expected. Any `h3` will not appear in the `Covered in this article` navigation but they should. Can I ask you to use `##` for all paragraphs we want to appear in `Covered in this article`?

Reply:
Can we also use section names such as `Navigating to XXX` instead of just `Navigation`